.build/build-checkstyle.xml

```diff
@@ -41,7 +41,7 @@
                     failOnViolation="true">
             <formatter type="plain"/>
             <formatter type="xml" tofile="${checkstyle.report.file}"/>
-            <fileset dir="${build.src.java}" includes="**/*.java"/>
+            <fileset dir="${build.src.java}" includes="**/*.java" excludes="build/**/*.java" />
```

Comment:
undo

Reply:
As far as I remember, checkstyle check was badly failing there with tons of warnings. Do the generated files also have to be validated?